test/integration/explorer-handler.spec.ts

```diff
@@ -161,141 +161,403 @@ testWithComponents(() => {
+    const { content, fetch, localFetch, theGraph, baseWearablesFetcher, contentServerUrl, alchemyNftFetcher } =
+      components
+    const baseWearables = generateBaseWearables(1)
+    const onChainWearables = generateWearables(1)
+    const thirdPartyWearables = generateThirdPartyWearables(1)
+    const entities = generateWearableEntities([
+      ...baseWearables.map((wearable) => wearable.urn),
+      ...onChainWearables.map((wearable) => wearable.urn),
+      ...thirdPartyWearables.map((wearable) => wearable.urn.decentraland)
+    ])
+
+    alchemyNftFetcher.getNFTsForOwner = jest
+      .fn()
+      .mockResolvedValue(thirdPartyWearables.map((wearable) => wearable.urn.decentraland))
+    baseWearablesFetcher.fetchOwnedElements = jest.fn().mockResolvedValue(baseWearables)
+    content.fetchEntitiesByPointers = jest.fn(async (pointers) =>
+      pointers.map((pointer) => entities.find((def) => def.id === pointer)).filter((e): e is Entity => e !== undefined)
+    )
+    theGraph.ethereumCollectionsSubgraph.query = jest.fn().mockResolvedValue({ nfts: onChainWearables.slice(0, 5) })
+    theGraph.maticCollectionsSubgraph.query = jest.fn().mockResolvedValue({ nfts: onChainWearables.slice(5, 10) })
+    fetch.fetch = jest.fn().mockImplementation((url) => {
+      if (url.includes('test-collection')) {
+        const thirdPartyEntities = entities.filter(entity =>
+          thirdPartyWearables.some(w => w.urn.decentraland === entity.id)
+        )
+        return {
+          ok: true,
+          json: () => ({
+            entities: thirdPartyEntities
+          })
+        }
+      } else {
+        return { ok: true, json: () => ({ entities: [] }) }
+      }
+    })
+
+    const wallet = generateRandomAddress()
+
+    // Test with trimmed=false
+    const r1 = await localFetch.fetch(`/explorer/${wallet}/wearables?trimmed=false`)
+    expect(r1.status).toBe(200)
+    const response1 = await r1.json()
+
+    // Test without trimmed parameter (should default to false)
+    const r2 = await localFetch.fetch(`/explorer/${wallet}/wearables`)
+    expect(r2.status).toBe(200)
+    const response2 = await r2.json()
+
+    // Both responses should be identical
+    expect(response1).toEqual(response2)
+
+    // Verify that all elements have the full entity structure
+    for (const element of response1.elements) {
+      expect(element).toHaveProperty('entity')
+
+      // Verify it's a full Entity (should have these properties)
+      expect(element.entity).toHaveProperty('id')
+      expect(element.entity).toHaveProperty('version')
+      expect(element.entity).toHaveProperty('type')
+      expect(element.entity).toHaveProperty('pointers')
+      expect(element.entity).toHaveProperty('timestamp')
+      expect(element.entity).toHaveProperty('content')
+      expect(element.entity).toHaveProperty('metadata')
+    }
+  })
+
+  it('handle trimmed parameter edge cases', async () => {
```

Comment:
can we update this test name to state which edge case are we willing to cover with it?